### PR TITLE
Get session for TMail

### PR DESCRIPTION
**Pull request by dab246**
- Get session when logged success

### CODE REVIEW

File: core/lib/data/network/config/endpoint.dart [hunk 1]
```diff
@@ -2,4 +2,5 @@ import 'package:core/data/network/config/service_path.dart';
 
 class Endpoint {
   static final ServicePath authentication = ServicePath('/api/login');
+  static final ServicePath session = ServicePath('/jmap/session');
```
Comment (chibenwa) [inline core/lib/data/network/config/endpoint.dart:5]: Pro-tip! Prefer `.well-known/jmap`

Comment (hoangdat) [reply core/lib/data/network/config/endpoint.dart:5]: thanks for your notice

Comment (chibenwa) [reply core/lib/data/network/config/endpoint.dart:5]: It allows auto discovery and redirects to jmap/session 

See https://jmap.io/spec-core.html#service-autodiscovery https://datatracker.ietf.org/doc/rfc8615/

File: lib/features/session/domain/usecases/get_session_interactor.dart [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import 'package:dartz/dartz.dart';
+import 'package:tmail_ui_user/features/session/domain/repository/session_repository.dart';
+import 'package:tmail_ui_user/features/session/domain/state/get_session_state.dart';
+
+class GetSessionInteractor {
+  final SessionRepository sessionRepository;
+
+  GetSessionInteractor(this.sessionRepository);
+
+  Future<Either<Failure, Success>> execute() async {
```
Comment (hoangdat) [inline lib/features/session/domain/usecases/get_session_interactor.dart:11]: Why not return a stream.
Loading -> State ...

Comment (dab246) [reply lib/features/session/domain/usecases/get_session_interactor.dart:11]: The processing of this task occurs on a loading screen. So I don't think the state needs to be concerned here.

**Review by hoangdat — APPROVED** [review]

**Pull request merged**